src/components/Accordion/index.js

```diff
@@ -0,0 +1,93 @@
+import React, { useState, useRef, useEffect } from 'react'
+import PropTypes from 'prop-types'
+import { genericPropTypes, genericPropsDefaults } from '../../utils/prop-types'
+import { Anchor } from '../Anchor'
+import {
+  AccordionBtn,
+  Content,
+  ContentContainer,
+  Icon,
+  StyledAccordion,
+} from './StyledAccordion'
+
+const Accordion = ({
+  a11yTitle,
+  children,
+  color,
+  text,
+  title,
+  onChange,
+  openByDefault,
+  ...rest
+}) => {
+  // set default open state to true
+  const [open, setOpen] = useState(openByDefault)
+  const contentRef = useRef(null)
+  let currentContent = null
+
+  // dynamically change the height of the content element
+  useEffect(() => {
+    currentContent.style.height = open
+      ? `${currentContent.scrollHeight}px`
+      : (currentContent.style.height = '0')
+  }, [contentRef, open])
+
+  return (
+    <StyledAccordion
+      forwardedAs="section"
+      aria-label={a11yTitle || title}
+      {...rest}
+    >
+      <AccordionBtn
+        forwardedAs="button"
+        aria-expanded={open}
+        direction="row"
+        align="center"
+        justify="flex-start"
+        onClick={() => {
+          setOpen(!open)
+          if (onChange) onChange()
+        }}
+      >
+        <Icon fillColor={color} isOpen={open} />
+        <Anchor color={color} margin="0" width="auto">
```

Comment:
Should this be an `Anchor` or a `Paragraph`? Just seems a little odd to me given that is it inside a `Button`. `Paragraph` appears to function OK.